tutorials/odoo-easy-deploy/index.mdx

```diff
@@ -185,5 +185,5 @@ The Odoo image [documentation](https://hub.docker.com/_/odoo) tells us that by m
 You can now start installing apps, configuring your settings and using Odoo. Check out the [official Odoo documentation](https://www.odoo.com/documentation/16.0/) for more help. 
 
 <Message type="note">
-  Please keep in mind that the Easy Deploy feature was designed to help you get started with Kubernetes: we do not recommend using this feature for production use for two reasons: First, because it requires your container image to be publicly available, i.e. without access restrictions. And second, because it does not allow you to do such things as add persistent volumes and link several containers, for example, without fine-tuning some settings yourself.
+  Keep in mind that the Easy Deploy feature was designed to help you get started with Kubernetes: we do not recommend using this feature for production use for two reasons: First, because it requires your container image to be publicly available, i.e. without access restrictions. And second, because it does not allow you to do such things as add persistent volumes and link several containers, for example, without fine-tuning some settings yourself.
```

Comment:
```suggestion
  Keep in mind that the easy deploy feature was designed to help you get started with Kubernetes: we do not recommend using this feature for production use for two reasons: First, because it requires your container image to be publicly available, i.e. without access restrictions. And second, because it does not allow you to do such things as add persistent volumes and link several containers, for example, without fine-tuning some settings yourself.
```